Tests/ScipioKitTests/ProjectGeneratorTests.swift

```diff
@@ -88,4 +87,63 @@ final class ProjectGeneratorTests: XCTestCase {
             )
         }
     }
+
+    func testGeneratedClangTarget() throws {
+        let (package, projectGenerator) = try prepareGenerator(for: clangPackagePath)
+        let projectPath = package.projectPath
+        try projectGenerator.generate()
+        XCTAssertTrue(fileSystem.exists(projectPath))
+
+        let project = try XcodeProj(pathString: projectPath.path)
+
+        // Check targets
+        let targets = project.pbxproj.nativeTargets
+        XCTAssertEqual(Set(targets.map(\.name)), ["some_lib"])
+
+        let target = try XCTUnwrap(project.pbxproj.targets(named: "some_lib").first)
+
+        // Check file tree
+        XCTAssertEqual(Set(project.pbxproj.groups.compactMap(\.name)), ["some_lib", "include", "Products"])
+        let rootGroup = try XCTUnwrap(project.pbxproj.rootGroup())
+        let libraryGroup = try XCTUnwrap(rootGroup.group(named: "some_lib"))
+        XCTAssertNotNil(libraryGroup.file(named: "some_lib.c"))
+
+        let includeGroup = try XCTUnwrap(libraryGroup.group(named: "include"))
+        XCTAssertNotNil(includeGroup.file(named: "some_lib.h"))
+
+        // Check build phase
+        let buildPhase = try XCTUnwrap(try target.sourcesBuildPhase())
+        XCTAssertEqual(buildPhase.files?.count, 1)
+
+        // Check header
+        let headerBuildPhase = try XCTUnwrap(target.buildPhases.first(where: { $0 is PBXHeadersBuildPhase }) as? PBXHeadersBuildPhase)
+        let publicHeader = try XCTUnwrap(headerBuildPhase.files?.first)
+        XCTAssertEqual(publicHeader.file?.path, "some_lib.h")
+        XCTAssertEqual(publicHeader.settings?["ATTRIBUTES"] as! [String], ["Public"])
+
+        // Check build settings
+        XCTAssertEqual(target.buildConfigurationList?.buildConfigurations.map(\.name), ["Debug", "Release"])
+        for configuration in target.buildConfigurationList!.buildConfigurations {
+            XCTAssertEqual(
+                configuration.buildSettings["MACH_O_TYPE"] as? String,
+                "staticlib", "If frameworkType is static, MACH_O_TYPE should be set"
+            )
+            XCTAssertEqual(
+                configuration.buildSettings["BUILD_LIBRARY_FOR_DISTRIBUTION"] as? String,
+                "YES"
+            )
+            XCTAssertEqual(
+                configuration.buildSettings["FRAMEWORK_SEARCH_PATHS"] as? [String],
+                ["$(inherited)", "$(PLATFORM_DIR)/Developer/Library/Frameworks"]
+            )
+            XCTAssertEqual(
+                configuration.buildSettings["CLANG_ENABLE_MODULES"] as? String,
+                "YES"
+            )
+            XCTAssertEqual(
+                configuration.buildSettings["DEFINES_MODULE"] as? String,
+                "YES"
+            )
```

Comment:
I feel it is better to have a test of `LD_RUNPATH_SEARCH_PATHS `

Reply:
Why do you think so compared with other settings?